pyensign/version.py

```diff
@@ -5,7 +5,7 @@
 __version_info__ = {
     "major": 0,
     "minor": 1,
-    "micro": 3,
+    "micro": 0,
```

Comment:
Our plan for SDK versioning is as follows:

1. The major and minor version numbers must follow the latest ensign version (e.g. right now Ensign is at v0.5.1; so the SDKs should be at v0.5.x) 
2. The micro (patch) number is independent for the SDK and can be incremented as many times as necessary. 

When we have our official "beta" release (not sure if it's this PR or not) then the version should be (v0.5.x).

Reply:
Cool, thanks for clarifying!